pygmt/datasets/tile_map.py

```diff
@@ -128,6 +141,7 @@ def load_tile_map(region, zoom="auto", source=None, lonlat=True, wait=0, max_ret
         ll=lonlat,
         wait=wait,
         max_retries=max_retries,
+        zoom_adjust=zoom_adjust,
```

Comment:
Might need to make this backwards compatible with `contextily<1.5.0`.

Reply:
Yes, need to check the contextily version and also better to mention that the `zoom_adjust` parameter requires contextily>=1.5.0.

Reply:
Ok, documented at e48c08df31b78a3cbc1b4f5baf18dbf10e7fc9d4, and added another commit at b428d12734f7a7d3386dfa9541627158de22ff44 to raise an error if `zoom_adjust` is used but `contextily<1.5.0` is installed.